sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_content_safety/_content_safety.py

```diff
@@ -61,7 +63,8 @@ class ContentSafetyEvaluator:
         }
     """
 
-    def __init__(self, credential, azure_ai_project: dict, parallel: bool = True):
+    def __init__(self, credential, azure_ai_project: dict, parallel: bool = True, eval_last_turn: bool = False):
```

Comment:
Shall we make `parallel` private? I think we might want to hide it from cx.

Reply:
Done.